src/main/kotlin/com/msd/event/application/dto/SpawnEventDTO.kt

```diff
@@ -0,0 +1,9 @@
+package com.msd.event.application.dto
+
+import java.util.*
+
+class SpawnEventDTO(
+    val robotId: UUID,
+    val playerId: UUID,
+    val otherSeeableRobots: List<UUID>
```

Comment:
Should be `otherVisibleRobots`

Reply:
yup, but that's the way it's specified in the API